lib/tests/streamlit/elements/vega_charts_test.py

```diff
@@ -162,6 +163,45 @@ def test_bad_theme(self):
             str(exc.exception),
         )
 
+    def test_works_with_element_replay(self):
+        """Test that it works with element replay if used as non-widget element."""
```

Comment:
nit: `"""Test that element replay works for vega if used as non-widget element"""`

Reply:
Updated